testing/regress/ecl-test

```diff
@@ -479,6 +484,16 @@ class RegressMain:
                 self.setup()
             elif self.args.func == 'run':
                 self.run()
+
+            #Process automatic deletion of workunits after test
+            if 'cleanup' in self.args:
+                cleanupMode = self.args.cleanup
+                if(cleanupMode == 'none'):
+                    logging.warning("Automatic cleanup of workunits has not been enabled.")       	    
+                else:
```

Comment:
The "cleanupMode" is already checked here. See my comment for the line 101 in cleanup.py.

Reply:
Noted